.github/workflows/lint-codeql.yaml

```diff
@@ -9,6 +9,7 @@ on:
 jobs:
   check_changes:
     name: Deduce required tests from code changes
+    if: github.repository == 'cilium/cilium' && needs.check_changes.outputs.go-changes == 'true'
```

Comment:
`needs.check_changes` won't work because the job can't have a dependency on itself.

Reply:
whoops, good catch. Got copy pasta happy.

Reply:
ammended.